ui/raidboss/data/07-dt/raid/r8s.ts

```diff
@@ -1554,7 +1566,7 @@ const triggerSet: TriggerSet<Data> = {
         const actor = actors[0];
```

Comment:
There is a bug in the original implementation of this trigger which means that it ***always*** calls the initial tether as SW, and therefore always calls the mechanic as SW -> SE.

I can't place this comment on the call to `getCombatants` just above here, but it's getting the source combatant which is the boss, instead of the target combatant of the player. 

![image](https://github.com/user-attachments/assets/ac3b0088-8690-4e18-b4ef-99a58cf2f91a)
![image](https://github.com/user-attachments/assets/3e3d4225-1dc3-4c24-9573-cd42296384d8)

Reply:
Ah, unfortunately I am away for another 20 days or so, hopefully someone else xan look into resolving the bug. I believe I resolved the merge conflict.